src/main/java/tutorlink/lists/StudentList.java

```diff
@@ -10,9 +9,10 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
-public class StudentList extends ItemList {
+public class StudentList {
     private static final String ERROR_DUPLICATE_MATRIC_NUMBER_ON_ADD = "Error! Student with Matric Number %s already"
             + "exists in the list!";
+    private static final String toStringDelimiter = "\n\t";
```

Comment:
Should the constant be TO_STRING_DELIMITER ?

Reply:
Good catch, thanks.